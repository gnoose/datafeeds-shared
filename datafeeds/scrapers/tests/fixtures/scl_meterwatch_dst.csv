Starting Date,Ending Date,Starting Time,Ending Time,kWh Cost,kWh Usage,kvarh
11/01/2020,11/01/2020,00:00:01,00:15:00,$8.41,121.920,75.360
11/01/2020,11/01/2020,00:15:01,00:30:00,$8.31,120.480,74.400
11/01/2020,11/01/2020,00:30:01,00:45:00,$8.28,120.000,74.400
11/01/2020,11/01/2020,00:45:01,01:00:00,$16.76,242.880,151.680
11/01/2020,11/01/2020,01:00:01,01:15:00,$16.73,242.400,149.760
11/01/2020,11/01/2020,01:15:01,01:30:00,$16.56,240.000,148.320
11/01/2020,11/01/2020,01:30:01,01:45:00,$16.53,239.520,149.760
11/01/2020,11/01/2020,01:45:01,02:00:00,$8.31,120.480,74.880
11/01/2020,11/01/2020,02:00:01,02:15:00,$8.25,119.520,74.400
11/01/2020,11/01/2020,02:15:01,02:30:00,$8.58,124.320,75.360
11/01/2020,11/01/2020,02:30:01,02:45:00,$8.21,119.040,73.920
11/01/2020,11/01/2020,02:45:01,03:00:00,$8.08,117.120,72.960
11/01/2020,11/01/2020,03:00:01,03:15:00,$8.11,117.600,72.480
11/01/2020,11/01/2020,03:15:01,03:30:00,$8.38,121.440,73.920
11/01/2020,11/01/2020,03:30:01,03:45:00,$9.27,134.400,75.360
11/01/2020,11/01/2020,03:45:01,04:00:00,$8.18,118.560,72.960
11/01/2020,11/01/2020,04:00:01,04:15:00,$8.05,116.640,72.000
11/01/2020,11/01/2020,04:15:01,04:30:00,$8.74,126.720,72.480
11/01/2020,11/01/2020,04:30:01,04:45:00,$8.05,116.640,71.520
11/01/2020,11/01/2020,04:45:01,05:00:00,$8.05,116.640,70.560
11/01/2020,11/01/2020,05:00:01,05:15:00,$8.94,129.600,72.480
11/01/2020,11/01/2020,05:15:01,05:30:00,$9.04,131.040,71.040
11/01/2020,11/01/2020,05:30:01,05:45:00,$9.14,132.480,71.520
11/01/2020,11/01/2020,05:45:01,06:00:00,$9.01,130.560,70.080
11/01/2020,11/01/2020,06:00:01,06:15:00,$9.04,131.040,69.600
11/01/2020,11/01/2020,06:15:01,06:30:00,$9.14,132.480,70.080
11/01/2020,11/01/2020,06:30:01,06:45:00,$9.14,132.480,69.120
11/01/2020,11/01/2020,06:45:01,07:00:00,$9.11,132.000,68.160
11/01/2020,11/01/2020,07:00:01,07:15:00,$9.14,132.480,68.160
11/01/2020,11/01/2020,07:15:01,07:30:00,$9.17,132.960,69.600
11/01/2020,11/01/2020,07:30:01,07:45:00,$9.21,133.440,68.640
11/01/2020,11/01/2020,07:45:01,08:00:00,$9.27,134.400,69.600
11/01/2020,11/01/2020,08:00:01,08:15:00,$9.44,136.800,71.520
11/01/2020,11/01/2020,08:15:01,08:30:00,$9.57,138.720,71.520
11/01/2020,11/01/2020,08:30:01,08:45:00,$9.64,139.680,72.000
11/01/2020,11/01/2020,08:45:01,09:00:00,$9.77,141.600,72.000
11/01/2020,11/01/2020,09:00:01,09:15:00,$9.77,141.600,71.520
11/01/2020,11/01/2020,09:15:01,09:30:00,$9.94,144.000,74.400
11/01/2020,11/01/2020,09:30:01,09:45:00,$9.87,143.040,72.960
11/01/2020,11/01/2020,09:45:01,10:00:00,$9.90,143.520,72.960
11/01/2020,11/01/2020,10:00:01,10:15:00,$9.94,144.000,73.920
11/01/2020,11/01/2020,10:15:01,10:30:00,$9.94,144.000,75.360
11/01/2020,11/01/2020,10:30:01,10:45:00,$10.00,144.960,75.360
11/01/2020,11/01/2020,10:45:01,11:00:00,$10.04,145.440,74.400
11/01/2020,11/01/2020,11:00:01,11:15:00,$10.07,145.920,75.360
11/01/2020,11/01/2020,11:15:01,11:30:00,$10.13,146.880,77.280
11/01/2020,11/01/2020,11:30:01,11:45:00,$10.27,148.800,78.240
11/01/2020,11/01/2020,11:45:01,12:00:00,$10.20,147.840,77.760
11/01/2020,11/01/2020,12:00:01,12:15:00,$10.30,149.280,78.240
11/01/2020,11/01/2020,12:15:01,12:30:00,$10.37,150.240,76.320
11/01/2020,11/01/2020,12:30:01,12:45:00,$10.40,150.720,78.720
11/01/2020,11/01/2020,12:45:01,13:00:00,$11.06,160.320,84.480
11/01/2020,11/01/2020,13:00:01,13:15:00,$12.52,181.440,88.800
11/01/2020,11/01/2020,13:15:01,13:30:00,$12.92,187.200,90.240
11/01/2020,11/01/2020,13:30:01,13:45:00,$12.88,186.720,89.280
11/01/2020,11/01/2020,13:45:01,14:00:00,$12.85,186.240,89.760
11/01/2020,11/01/2020,14:00:01,14:15:00,$12.98,188.160,91.680
11/01/2020,11/01/2020,14:15:01,14:30:00,$12.09,175.200,89.280
11/01/2020,11/01/2020,14:30:01,14:45:00,$11.23,162.720,87.360
11/01/2020,11/01/2020,14:45:01,15:00:00,$11.63,168.480,87.840
11/01/2020,11/01/2020,15:00:01,15:15:00,$12.39,179.520,89.760
11/01/2020,11/01/2020,15:15:01,15:30:00,$12.49,180.960,90.720
11/01/2020,11/01/2020,15:30:01,15:45:00,$12.55,181.920,88.800
11/01/2020,11/01/2020,15:45:01,16:00:00,$12.75,184.800,90.240
11/01/2020,11/01/2020,16:00:01,16:15:00,$12.65,183.360,88.320
11/01/2020,11/01/2020,16:15:01,16:30:00,$12.65,183.360,87.360
11/01/2020,11/01/2020,16:30:01,16:45:00,$12.25,177.600,84.960
11/01/2020,11/01/2020,16:45:01,17:00:00,$11.82,171.360,83.040
11/01/2020,11/01/2020,17:00:01,17:15:00,$11.46,166.080,79.680
11/01/2020,11/01/2020,17:15:01,17:30:00,$11.23,162.720,79.200
11/01/2020,11/01/2020,17:30:01,17:45:00,$11.16,161.760,78.240
11/01/2020,11/01/2020,17:45:01,18:00:00,$11.16,161.760,77.760
11/01/2020,11/01/2020,18:00:01,18:15:00,$11.03,159.840,77.760
11/01/2020,11/01/2020,18:15:01,18:30:00,$10.90,157.920,77.760
11/01/2020,11/01/2020,18:30:01,18:45:00,$10.83,156.960,76.320
11/01/2020,11/01/2020,18:45:01,19:00:00,$10.73,155.520,76.320
11/01/2020,11/01/2020,19:00:01,19:15:00,$9.80,142.080,75.360
11/01/2020,11/01/2020,19:15:01,19:30:00,$8.68,125.760,73.440
11/01/2020,11/01/2020,19:30:01,19:45:00,$8.68,125.760,74.880
11/01/2020,11/01/2020,19:45:01,20:00:00,$8.64,125.280,74.880
11/01/2020,11/01/2020,20:00:01,20:15:00,$8.71,126.240,75.840
11/01/2020,11/01/2020,20:15:01,20:30:00,$8.68,125.760,74.400
11/01/2020,11/01/2020,20:30:01,20:45:00,$9.01,130.560,71.040
11/01/2020,11/01/2020,20:45:01,21:00:00,$8.18,118.560,68.160
11/01/2020,11/01/2020,21:00:01,21:15:00,$8.08,117.120,68.160
11/01/2020,11/01/2020,21:15:01,21:30:00,$9.11,132.000,72.000
11/01/2020,11/01/2020,21:30:01,21:45:00,$9.14,132.480,73.920
11/01/2020,11/01/2020,21:45:01,22:00:00,$8.61,124.800,72.960
11/01/2020,11/01/2020,22:00:01,22:15:00,$8.08,117.120,72.480
11/01/2020,11/01/2020,22:15:01,22:30:00,$8.08,117.120,74.400
11/01/2020,11/01/2020,22:30:01,22:45:00,$8.08,117.120,73.440
11/01/2020,11/01/2020,22:45:01,23:00:00,$8.02,116.160,72.480
11/01/2020,11/01/2020,23:00:01,23:15:00,$8.94,129.600,74.400
11/01/2020,11/01/2020,23:15:01,23:30:00,$9.04,131.040,73.920
11/01/2020,11/01/2020,23:30:01,23:45:00,$9.07,131.520,75.360
11/01/2020,11/02/2020,23:45:01,00:00:00,$8.94,129.600,73.920

