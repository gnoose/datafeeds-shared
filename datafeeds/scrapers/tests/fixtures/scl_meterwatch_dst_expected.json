{"2020-11-01": [487.68, 481.92, 480.0, 485.76, 484.8, 480.0, 479.04, 481.92, 478.08, 497.28, 476.16, 468.48, 470.4, 485.76, 537.6, 474.24, 466.56, 506.88, 466.56, 466.56, 518.4, 524.16, 529.92, 522.24, 524.16, 529.92, 529.92, 528.0, 529.92, 531.84, 533.76, 537.6, 547.2, 554.88, 558.72, 566.4, 566.4, 576.0, 572.16, 574.08, 576.0, 576.0, 579.84, 581.76, 583.68, 587.52, 595.2, 591.36, 597.12, 600.96, 602.88, 641.28, 725.76, 748.8, 746.88, 744.96, 752.64, 700.8, 650.88, 673.92, 718.08, 723.84, 727.68, 739.2, 733.44, 733.44, 710.4, 685.44, 664.32, 650.88, 647.04, 647.04, 639.36, 631.68, 627.84, 622.08, 568.32, 503.04, 503.04, 501.12, 504.96, 503.04, 522.24, 474.24, 468.48, 528.0, 529.92, 499.2, 468.48, 468.48, 468.48, 464.64, 518.4, 524.16, 526.08, 518.4]}
